modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/css/_general.scss

```diff
@@ -219,6 +219,11 @@ td.lfr-name-column {
 
 .form-builder-sidebar,
 .portlet-forms {
+	.forms-bar-ie {
```

Comment:
@marcosapmf, does this class aim to modify which bar? The navigation bar or the management bar? Would be important make it clear on the class name.

Reply:
@natocesarrego this class aims to modify both of navigation and management bar. That's why I named it with a more generic name